core/src/main/resources/hudson/Messages_it.properties

```diff
@@ -1,22 +1,22 @@
 FilePath.did_not_manage_to_validate_may_be_too_sl=Impossibile validare {0} (potrebbe essere troppo lento)
 FilePath.validateAntFileMask.whitespaceSeparator=\
-  Non � pi� possibile utilizzare spazi bianchi come separatore. Si utilizzi invece "," come separatore.
+  Non � pi� possibile utilizzare spazi bianchi come separatore. Si utilizzi invece "," come separatore.
 FilePath.validateAntFileMask.doesntMatchAndSuggest=\
-  Non vi sono corrispondenze per "{0}", ma per "{1}" s�. Forse era ci� che si intendeva?
-FilePath.validateAntFileMask.portionMatchAndSuggest=Non vi sono corrispondenze per "{0}", bench� "{1}" esista
+  Non vi sono corrispondenze per "{0}", ma per "{1}" s�. Forse era ci� che si intendeva?
```

Comment:
Not sure if this is mojibake or if GitHub just is not detecting encoding properly, but anyway this looks like an unintentional change.

Reply:
I think you are looking at an outdated view, it doesn't look like that for me

Reply:
Yeah, probably just GH showing me an intermediate commit that was later amended.